index.js

```diff
@@ -281,6 +298,30 @@ AFRAME.registerComponent('super-hands', {
       }
     }
   },
+  onActivateStartButton: function (evt) {
+    const carried = this.state.get(this.GRAB_EVENT)
+    let activated = this.state.get(this.ACTIVATE_EVENT)
+    if (carried) {
+      if (!activated && !this.emitCancelable(carried, this.ACTIVATE_EVENT, {hand: this.el, buttonEvent: evt})) {
```

Comment:
This would mean only an entity which is already the grabbed entity can be activated. In this respect, it wouldn't be much different from the drag gesture. What would you say to making this a more general gesture, just the act of pressing a button while targeting an entity? I'm thinking this way the `activatable` component would replace the `clickable` component and do a better job of what it has been trying to do without interfering with grabbing. When you do want an activation that is dependent on also being grabbed, that logic could be handled within the reaction component instead.

Reply:
This was definitely intended to be similar to `draggable`, but more generic and customizable (since you can specify exactly what state to set on the objects so you can have multiple). Also, given `clickable` exists, I didn't see the need to allow `activatable` to work on ungrabbed objects. I am not opposed to rolling all three of these together since they are pretty similar.